tests/functional/test_base_cpu.py

```diff
@@ -20,7 +20,6 @@ def test_base_cpp(tester, image, install_openvino_dependencies):
     }
     tester.test_docker_image(
         image,
-        [install_openvino_dependencies,
```

Comment:
add mark < 2021.3 to test and create one more test for >= 2021.3

Reply:
what is the difference between them?